### PR TITLE
Support bytes server values. Prefer decimal prefixes for memory

### PR DESCRIPTION
**Pull request by romasku**
Closes https://github.com/neuro-inc/neuro-cli/issues/2730

### CODE REVIEW

File: neuro-cli/tests/unit/formatters/ascii/TestJobOutputFormatter.test_job_on_preemptible_node[human]_0.ref [hunk 1]
```diff
@@ -8,8 +8,8 @@ Job                     test-job
  Priority                Normal                
  Price (credits / hour)  15.0000               
  Current cost            150.0000              
- Resources                Memory  16.0M        
-                          CPU       0.1        
+ Resources                Memory  16 Bytes
```
Comment (serhiy-storchaka) [inline neuro-cli/tests/unit/formatters/ascii/TestJobOutputFormatter.test_job_on_preemptible_node[human]_0.ref:11]: Isn't 16 bytes too low?

File: neuro-cli/tests/unit/formatters/test_jobs_formatters.py [hunk 2]
```diff
@@ -1463,7 +1463,9 @@ def test_format_telemetry_no_gpu(
             datetime_formatter=datetime_formatter,
         ) as fmt:
             timestamp = 1_517_248_466.238_723_6
-            telemetry = JobTelemetry(cpu=0.12345, memory=256.123, timestamp=timestamp)
+            telemetry = JobTelemetry(
+                cpu=0.12345, memory_bytes=int(256.123 * 2**20), timestamp=timestamp
```
Comment (serhiy-storchaka) [inline neuro-cli/tests/unit/formatters/test_jobs_formatters.py:1467]: Just `268564430`?

File: neuro-cli/src/neuro_cli/utils.py [hunk 3]
```diff
@@ -616,7 +616,7 @@ def parse_permission_action(action: str) -> Action:
 def format_size(value: Optional[float]) -> str:
     if value is None:
         return ""
-    return humanize.naturalsize(value, gnu=True)
+    return humanize.naturalsize(value)
```
Comment (serhiy-storchaka) [inline neuro-cli/src/neuro_cli/utils.py:619]: This change is responsible for many test changes, and is not essential for functionality. I would make it in a separate PR.

File: neuro-cli/src/neuro_cli/parse_utils.py [hunk 4]
```diff
@@ -28,16 +28,42 @@ def parse_memory(memory: str) -> int:
     returns value in bytes"""
 
     # Mega, Giga, Tera, etc
-    prefixes = "MGTPEZY"
+
+    prefix_to_factor = {
+        "": 1,
+        # Binary
+        "ki": 2**10,
```
[the remark this answers is not in the document]
Comment (romasku) [reply neuro-cli/src/neuro_cli/parse_utils.py:35]: Got it, thank you, will fix it.

File: neuro-cli/src/neuro_cli/utils.py (hunk 3 above)
Comment (romasku) [reply neuro-cli/src/neuro_cli/utils.py]: Good point, I've reverted this change.

**Review by asvetlov — APPROVED** [review]

**Review by asvetlov — COMMENTED** [review]
Add a changelog please
